CI/travis/make_linux

```diff
@@ -5,6 +5,11 @@ if [ "x${COVERITY_SCAN_PROJECT_NAME}" != "x" ] ; then exit 0; fi
 . CI/travis/lib.sh
 
 handle_default() {
+	# make sure we are up to date
+	if [ "$LDIST" = "DO_NOT_DEPLOY" ] ; then
+		./CI/travis/check_kernel.sh
+	fi
```

Comment:
oh right; 1 more thing; i kept forgetting to mention this here;
would it be worth to move this to the end of the `handle_default()` function?

it would be an idea to let the compiler have a run first;
those errors would have a slightly higher priority [for this build];

Reply:
fixed